### PR TITLE
[DEVX-1286][DEVX-1254] Add reference for filters in TestCafe config + Update cloud versions

### PR DESCRIPTION
**Pull request by FriggaHel**
### Description

Adding filters for testcafe

### Motivation and Context

New feature soon available.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/testrunner-toolkit/configuration/testcafe.md [hunk 1]
```diff
@@ -533,6 +533,87 @@ The explicit name, file glob, or location of the test files to be included in th
...
+
+#### `fixtureMeta`
+<p><small>| OPTIONAL | OBJECT |</small></p>
+
+Runs tests whose fixture’s metadata matches the specified key-value pair.
+
+```yaml
+  fixtureMeta:
+    env: staging
+```
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/configuration/testcafe.md:613]:
```suggestion
Runs any tests included in fixtures whose metadata matches the specified key-value pairs. Accepts one or more key-value definitions. If multiple pairs are specified, matching tests must contain all of the specified metadata values.

```yaml
filter:
  fixtureMeta:
    env: staging
```
```

File: docs/web-apps/automated-testing/testcafe.md [hunk 2]
```diff
@@ -121,7 +121,7 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
  |TestCafe Version|Supported Browsers|
  |----|----|
- |1.16.1|Please see [release notes](https://github.com/saucelabs/sauce-testcafe-runner/releases/tag/v0.11.0)|
+ |1.16.1|Please see [release notes](https://github.com/saucelabs/sauce-testcafe-runner/releases/tag/v0.12.0)|
```
Comment (sweeneyskirt-sl) [inline docs/web-apps/automated-testing/testcafe.md:124]: Could you please also update the docker supported versions table on the main saucectl page: /docs/testrunner-toolkit.md? Thanks!

File: docs/testrunner-toolkit.md [hunk 3]
```diff
@@ -237,7 +254,7 @@ Browser support for each framework is based on the Sauce Labs docker images prov
 
 |TestCafe Version|Supported Browsers|
 |----|----|
-|1.16.1|Please see [release notes](https://github.com/saucelabs/sauce-testcafe-runner/releases/tag/v0.11.0)|
+|1.16.1|Please see [release notes](https://github.com/saucelabs/sauce-testcafe-runner/releases/tag/v0.12.0)|
```
Comment (alexplischke) [inline docs/testrunner-toolkit.md:257]:
```suggestion
|1.16.1|Please see [release notes](https://github.com/saucelabs/sauce-testcafe-runner/releases/tag/v0.12.1)|
```

File: docs/web-apps/automated-testing/testcafe.md [hunk 4]
```diff
@@ -121,7 +136,7 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
```
Comment (alexplischke) [inline docs/web-apps/automated-testing/testcafe.md:139]:
```suggestion
 |1.16.1|Please see [release notes](https://github.com/saucelabs/sauce-testcafe-runner/releases/tag/v0.12.1)|
```

**Pull request merged**